.github/workflows/tests.yml

```diff
@@ -147,7 +147,7 @@ jobs:
 
           # bin/max (sizes are in MiB)
           SIZES="lxc 15
-                 lxd-agent 12"
+                 lxd-agent 13"
```

Comment:
this should be a warning sign we've accidentally imported something we shouldn't.